content/post/rust-error-handling.md

```diff
@@ -1549,12 +1549,18 @@ enum CliError {
 }
 {{< /high >}}
 
-And add a new `From` impl:
+To reflect this change we need to update the previous `impl From<num::ParseIntError> for CliError` and add the new `impl From<num::ParseFloatError> for CliError`:
```

Comment:
Sorry to keep bothering you with nits, but could you make sure the text is formatted in a style consistent with the surrounding text? In this case, please wrap to 79 columns inclusive. :-) Thanks.

Reply:
No problem, I treat things surrounded by backticks as one word.